spec/controllers/course/controller_spec.rb

```diff
@@ -64,5 +50,28 @@ def publicly_accessible?
         end
       end
     end
+
+    describe '#current_component_host' do
+      it 'returns the current course' do
+        allow(controller).to receive(:current_course).and_return(course)
+        expect(controller.current_component_host).to be_a Course::ComponentHost
+      end
+    end
+
+    describe '#sidebar' do
+      it 'returns an empty array when no components included' do
+        allow(controller).to receive_message_chain('current_component_host.components')
+                               .and_return([])
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 23) spaces for indenting an expression spanning multiple lines.